animal.go

```diff
@@ -32,9 +32,11 @@ func GetUserYesOrNo(question string) string {
 	scanner := bufio.NewScanner(os.Stdin)
 	scanner.Scan()
 	input := scanner.Text()
-	fmt.Printf("You typed: %q", input)
+	fmt.Printf("You typed: %s\n", input)
```

Comment:
Let's take these print statements out, since libraries in general shouldn't print anything to the terminal, and we should be able to tell from the test results whether or not the code is working!

Reply:
Print statements have been removed.